Assets/Plugins/Source/Core/EOSManager.cs

```diff
@@ -1640,13 +1658,20 @@ public void OnApplicationShutdown()
                     Debug.Log("Releasing the EOS Platform Interface.");
                     GetEOSPlatformInterface()?.Release();
 
+#if EOS_UNLOAD_SDK_ON_SHUTDOWN
                     Debug.Log("Shutting down the platform interface.");
                     ShutdownPlatformInterface();
+#endif
+
                     SetEOSPlatformInterface(null);
+
+
 #endif
 #if UNITY_EDITOR
+#if EOS_UNLOAD_SDK_ON_SHUTDOWN
```

Comment:
This relates to a previous comment; here, you do put the `UnloadAllLibraries` inside the `#if EOS_UNLOAD_SDK_ON_SHUTDOWN`. Should it be called above?

Reply:
In light of making the code less confusing, I'm going to change the code above.